eda-env/Lib/site-packages/pandas/tests/window/test_rolling.py

```
@@ -0,0 +1,1979 @@
+        ),
+    ],
+)
+def test_iter_rolling_datetime(expected, expected_index, window):
+    # GH 11704
+    ser = Series(range(5), index=date_range(start="2020-01-01", periods=5, freq="D"))
+
+    expected = [
+        Series(values, index=idx) for (values, idx) in zip(expected, expected_index)
+    ]
+
+    for expected, actual in zip(expected, ser.rolling(window)):
```

Ruff (0.11.9)

Loop control variable `expected` overrides iterable it iterates

(B020)